### DISCUSSION TITLE
m-bus master board is getting hot, no 30V

### DISCUSSION THREAD

**Discussion by kusmi** [Zeppelin500/MBusino#18] [General]
Not sure if AliExpress-seller changed m-bus board manufacturer, as mine looks a bit different (no arrows on PCB, but similar parts - except the inductor seems to be 10 µH instead of 15 µH as in your board?) - but I can't seem to get it to work.

If I connect it with my S2 Mini I don't see 30V output, only a few volt (fluctuating 1-4 V - I have to analyze with scope if it ever goes to 30 V - perhaps my multimeter is too slow, but I though m-bus only fluctuates around 30 V and not down to 4 V) and the area around the conductor gets very hot (too hot to touch).

Funnily I already ordered a replacement m-bus board from different seller - looks 100% identical with my previous, but this shows the same behavior: Missing 30 V voltage, too hot to touch.

Anybody with similar experience or knows what to check/adjust?

![IMG_4118](https://github.com/user-attachments/assets/39fd60f3-431f-4229-84fe-7bc2619056d5)
![IMG_4119](https://github.com/user-attachments/assets/1892cf7b-266a-4386-a007-9419ce1a0dfc)
![IMG_4121](https://github.com/user-attachments/assets/4e515f46-9bb5-48aa-b864-96966a9f0d36)

**Comment by Zeppelin500** [comment]
Hallo kusmi,

nein, das hat noch keiner berichtet. Wo hast du es gekauft? Überprüfe nochmal dein wiring. Was hängt am M-Bus?
Hab hier noch ein Ersatzboard, hab ich ca. vor 6 Monaten gekauft. Hat auch diese Pfeile. Was steht auf deiner Rückseite? Hier die Fotos.
Bitte um Rückmeldung.
![IMG_20240730_184637](https://github.com/user-attachments/assets/e4fcb26a-724a-4069-94d1-466f91453277)
![IMG_20240730_184627](https://github.com/user-attachments/assets/470f7bb8-013b-4683-a6cf-0a15acc64cda)

**Comment by kusmi** [comment]
Merci für Deine Rückmeldung!

Ich hab es ursprünglich hier gekauft: https://de.aliexpress.com/item/1005003717810499.html

da wusste ich noch nichts von Deinem Projekt. Nach dem Fehlschlag hab ich das Board auch über Deinen Link gekauft: https://de.aliexpress.com/item/33008746192.html

und im beiden Fällen das gleiche Board bekommen (wie auf Foto) sogar gewisse Komponenten wurden gleichschräg aufgelötet 😉

Ich hab Dir noch die Rückseite fotografiert.

Hab auch mal versucht ein paar Stützkondensatoren 100 uF und 10 uF an vin und vcc-ttl anzulegen, da es zum starten des step-ip converter ev. etwas mehr Strom braucht als der s2 liefern kann - half aber nicht.

Sonst hab ich an den m-bus noch kein Gerät (Wasserzähler) angeschlossen, da die Spannung ja nicht stimmte - oder darf der m-bus nicht floaten?

Und auch einen weiteren S2 mini mal versucht. Ich kuck noch in der Bastelkiste ob ich einen 8266er finde.

![IMG_4126](https://github.com/user-attachments/assets/d2858d65-00bc-442b-a91b-a023e6d0ebd4)

**Comment by Zeppelin500** [comment]
Die 5V kommen ja nicht vom S2 Mini, sondern von deinem Netzteil. Im ESP ist es nur durch geschleift. 
Ist beim 8266 genau so. Tausch mal das Netzteil. 
Ich weiß von 70 erfolgreichen Implementierungen. Bis jetzt hat keiner das Problem gemeldet. Allerdings ist es seit dem Frühjahr ruhig geworden, da die meisten Wärmelängenzähler auslesen und das Thema im Sommer keinen Focus hat :-)

**Comment by kusmi** [comment]
Das ist ein guter Punkt, hatte immer das gleiche USB Netzteil… leider keine Änderung wenn ich mit zwei anderen (und auch andere usb-Kabel) probiere.

ESP läuft ja auch und Spannung stimmt auch wenn ich messe - ist ja ganz übersichtlich mit gnd, 5V, 3.3V und den rx/tx (die ich auch schon mal abgehängt hatte, mal auf gnd und mal auf 3.3v gesetzt hatte am ursprünglichen/älteren m-bus board)

**Comment by kurt-k** [comment]
Habe Ende letzter Woche das gleiche Board bekommen wie du und bin über das gleiche Problem gestolpert.
Dachte erst ich hätte meine Platine falsch gelötet aber nachdem ich das ganze nochmal auf dem Steckbrett aufgebaut habe, gleiche Ergebnis.

Die Spule wird sehr sehr heiß und keine 30V am Ausgang.
Eingespeist habe ich es mit 5V über ein Labornetzteil. Solange der TTL VCC in der Luft hängt, ist die Stromaufnahme bei 0mA. Mit 5V oder 3,3V auf TTL VCC springt die Stromaufnahme auf 350mA und die Spule wird heiß.

Gekauft übrigens über den Link hier aus dem Projekt.

**Comment by Zeppelin500** [comment]
Mhh, wäre ärgerlich wenn das jetzt nicht mehr funktioniert. Hersteller ist auf jeden Fall ein anderer jetzt.
Hab jetzt mal mein Live System gemessen. Ob an M-Bus was angeschlossen ist, oder nicht, macht keinen Unterschied mit dem Multimeter. ~29V.
Du könntest noch probieren z.B. 12V einzuspeisen anstatt der 5V. Masse verbinden.
Falls das klappt ist der DC/DC Wandler überfordert mit den 5V.

**Comment by kurt-k** [comment]
Hab es gerade nochmal mit 12V Einspeisung für die MBus-Platine getestet, leider ohne Erfolg.
Die Leistungsaufnahme bleibt konstant und die Spule erhitzt sich sehr stark. 
Am Ausgang bleibt die Spannung trotzdem bei 0V.

**Comment by Zeppelin500** [comment]
OK, dann bin ich mit meinem Latein am Ende.
Hab nochmal nachgesehen. Meine Spule bleibt absolut kalt im Betrieb.
Bitte reklamiert das. Es war schonmal ein Master defekt, der Austausch war unkompliziert und schnell. Eine Lötbrücke seht ihr nicht?
Hab mir jetzt auch mal einen Master bestellt...

**Comment by kusmi** [comment]
Ich hab mal alle Bauteile verglichen - das einzige offensichtliche ist die Spule die neu nur 10 µF ist statt bei Dir 15 µF. Sonst gibt es noch andere Änderungen, die ich aber nicht einschätzen kann:

- Spule 10 µF statt 15 µF
- Ein Bauteil ist signifikant kleiner (heisst bei mir "AV", bei Dir "GK C49 AV")
- Das daneben heisst "AG R3W" bei Dir "AG QK"
- Dann sind alle 702 P (Mosfets?) bei Dir "72K"
- Beim M-Bus Anschluss "84118 A3278" bei Dir "84118 4261"

PS: Ein paar Wiederstände scheinen allenfalls auch geändert zu sein, kann das nicht alles entziffern...

Ich hab mir nochmals bei AliExpress ein Board bestellt dass laut Beschreibung auch diese "Pfeile" auf der Platine hat - mal schauen was ich dort bekomme 😄 : https://de.aliexpress.com/item/1005005856013337.html - melde mich dann wieder

**Comment by kurt-k** [comment]
@kusmi genau das Board habe ich Zuge der Rückgabe eben auch direkt bestellt. 

Habe im laufenden Betrieb auch noch die Spannung an allen Kondensatoren gemessen um zu prüfen ob die 30V einfach nur nicht durchgeschaltet werden, aber leider auch ohne Erfolg. An keinem Kondensator lag eine signifikante Spannung an

**Comment by Zeppelin500** [comment]
Mit den Beschriftungen kann ich auch nichts Anfangen. Denke da ist ein [MT3608](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.olimex.com/Products/Breadboarding/BB-PWR-3608/resources/MT3608.pdf&ved=2ahUKEwiO4JD6idGHAxXSA9sEHUuCHs0QFnoECCAQAQ&usg=AOvVaw0KhDotLTD6GhfvWaSXtCgb) oder sowas verbaut.

**Comment by Zeppelin500** [comment]
So, habe nun das selbe Board von DykbRadio erhalten. Keine Funktion und die Spule wird heiß :-(
So ein Misst. Werde reklamieren. Bitte macht das auch. Ich hoffe, sie geben es an den Hersteller weiter, damit da eine Analyse laufen kann.

**Comment by kusmi** [comment]
Ich hab reklamiert - aber AliExpress war etwas umständlich. Hatte ehrlich gesagt auch noch nie eine „Defekt-Rückgabe“ nur „Nicht erhalten Rückgaben“

Musste allerhand Beweise liefern mit Fotos etc.

oder wie macht ihr das bei AliExpress?

**Reply by kurt-k** [reply]
Ich habe meine Platine auch reklamiert.
Nachdem ich einfach nur ein Foto der Platine gemacht habe, wurde ein Hermes Rücksendeetikett erstellt.
In der Sekunde der Einlieferung bekam ich dann schon die Mail das mein Geld zurückerstattet wird.

**Reply by kusmi** [reply]
Das hab ich auch gemacht und Link hier auf die Diskussion. Wollten aber noch mehr Infos und ich hab dann noch Fotos mit Voltmeter und Temperatur gemacht. Die Hitze kommt vom step-up chip und hatte mindestens 83 °C bei mir.

**Comment by Zeppelin500** [comment]
selbst hab ich auch noch nie reklamiert. Hatte nur mal von Einem, der ein defektes bekommen hat, die Rückmeldung bekommen, dass es kein Problem war.

Bin grad dabei, mal sehen...

**Comment by Zeppelin500** [comment]
Hab jetzt mal den Lieferanten direkt angeschrieben und ihn gebeten den Bestand zu sperren, den Hersteller zu informieren und wenn möglich wieder auf "np-Link" zu schwenken.

**Comment by kurt-k** [comment]
Die Bestellung des Links von oben ist gerade angekommen. Leider ist nicht die abgebildete Platine mit den Pfeilen sondern wieder das neuste Modell gekommen. Werde sie sicherheitshalber einmal durchmessen aber ich mache mir wenig Hoffnung.

Parallel bestelle ich mir jetzt noch eine andere Mbus-Platine die ich bei AliExpress gesehen habe. Hier sind die Anschlüsse mit Winkeleligen Stecker an der langen Seite ausgeführt. Ich werde berichten.

**Reply by kurt-k** [reply]
Nächster Versuch:
https://de.aliexpress.com/item/1005006293773527.html

**Reply by kurt-k** [reply]
Die heute gelieferte Platine zeigt leider wie zu erwarten das gleiche Fehlerbild und geht direkt wieder zurück.

**Reply by kusmi** [reply]
Auweia… meine ist noch nicht gekommen - aber dann werd ich wohl die 3. falsche bekommen haben :-)

**Comment by Zeppelin500** [comment]
Ihr könnt auch die Master von Zihatec verwenden.
Die laufen einwandfrei. Zu beachten ist nur, dass er ein extra Netzteil braucht, da nicht 5V kompatibel und er sich nur mit dem S2 Mini versteht, den 8266 mag er nicht.

**Comment by Zeppelin500** [comment]
Ein kleines Update von mir.
Leider bekomme ich mit dem Händler keine vernünftige Kommunikation hin. Hab langsam die Hoffnung verloren. Er Antwortet zwar, aber nicht auf meine Anliegen :-)

Dafür gibt es andere gute Neuigkeiten.
Stehe mit dem Entwickler von Zihatec in Kontakt. Der Master für den Raspi Pico ist bereits 5V kompatibel über eine Lötbrücke. Hatte den Wunsch bereits vor ein paar Monaten geäußert und er hat es umgesetzt --> funktioniert ohne extra Netzteil. Passt so allerdings nicht ins MBusino Gehäuse.
Aber noch besser, er wird, extra für uns, ein 5V kompatibles, integrierbares Masterboard veröffentlichen. 
Demnach werde ich dann einen neuen MBusino mit seinem Master veröffentlichen.

**Reply by Zeppelin500** [reply]
Update:
ich habe von Zihatec Muster ihres MBus Masters bekommen. Fliegend verdrahtet läuft das schon mal. Eine MBusino Version für den Zihatec Master ist gezeichnet und Muster bestellt.

![1725361606608](https://github.com/user-attachments/assets/d58bcb52-ee68-41a0-84d4-289ccca40e64)

**Reply by alfere2** [reply]
Hi, weißt Du, ob/wann dieser Mbus-Master verkauft werden soll?
Wollte Deinen verlinkten von Dykbvell auf AliExpress bestellen, der ist aber aktuell nicht verfügbar.

![image](https://github.com/user-attachments/assets/6d08cb56-42fa-44b4-aa59-311fd2cf0849)

**Reply by Zeppelin500** [reply]
Vielleicht ein gutes Zeichen, dass er überall nicht lieferbar ist. Die vielen Reklamationen haben wohl geholfen. Hoffentlich gibt es bald ein neues, funktionierendes Los.

In meinem neuen MBusino Muster für Zihatec Master war ein Fehler, demnach neu gezeichnet und neu bestellt. In 2 Wochen kann ich mehr sagen.

**Reply by Zeppelin500** [reply]
Falls du es noch nicht auf der Hauptseite gesehen hast. MBusino "Edition Z" und der Zihatec Master sind verfügbar.

**Comment by kurt-k** [comment]
Ich kann seit heute Erfolg vermelden. Das MBus Board mit den gewinkelten Stecker welches ich oben gepostet habe funktioniert.
![PXL_20240814_115204075](https://github.com/user-attachments/assets/297824f8-a269-471e-a49b-c31208b30065)
![PXL_20240814_115155227](https://github.com/user-attachments/assets/886eeebb-791e-47c1-9ef9-e18ed1e2b4ac)

**Reply by kusmi** [reply]
Supper news! Sollte bei mir auch bald ankommen!

**Reply by kusmi** [reply]
Mit diesem Board (gestern gekommen) klappt es im Prinzip nun auch - muss jetzt noch Debuggen, da mein Wasserzähler oder die M-Bus-Library momentan noch "Stuss" liefert 😄 über die Zeit füllen sich alle möglichen Felder mit irrsinnigen Werten. Sieht sonst fast so aus, als wäre das Ding an einem Kernkraftwerk direkt angeschlossen...

<img width="372" alt="Screenshot 2024-08-16 at 11 13 26" src="https://github.com/user-attachments/assets/ae4ec858-113d-4f45-a63a-462d0f241efe">

**Reply by Zeppelin500** [reply]
Das ist kein Problem, das kann ich fixen. Gibt es von deinem Zähler ein Beispieltelegramm?
Schreib mir bitte eine Mail.

**Reply by kusmi** [reply]
Merci für das Angebot - ich muss zuerst mal eine Beispielübertragung des 'Sensus iPerl' abgreifen - der Hersteller hat mir da keine Doku geben können (ausser dass die Kommunikation 2400 Baud wäre - Standard für M-Bus).

**Reply by Zeppelin500** [reply]
Einfach Zeile 416 und 424 löschen und mit dem MQTT Explorer nachsehen.

Den Fix bekomme ich auch ohne Doku hin mit deiner Hilfe. Brauche nur dass Roh Telegramm. Das sollten wir aber nicht hier unter diesem Faden machen.

**Reply by mrklsr** [reply]
Ich habe mir das gleiche MBus Board mit den gewinkelten Steckern besorgt.
Welche Spannungsversorgung legst du denn bei V_IN an? Ich messe an MBUS+ und MBUS - nur knapp 18 V, an V_H zu GND allerdings 28 V

**Reply by Zeppelin500** [reply]
5V. Bei mir schafften die CN Master immer so 30V auf dem M-Bus

**Comment by tom199-199** [comment]
Hallo,

habe soeben das am 18.08.2024 bestellte Board von Dykbcells über AliExpress bekommen.

https://de.aliexpress.com/item/1005003717810499.html

Kann es gerade nur optisch vergleichen, aber es sieht wieder nach der ursprünglichen Version aus (15 uH, Pfeile auf der Platine,...).



Auch die anderen Werte wie in der  Diskussion beschrieben sind wieder die "alten" Werte



Spule 10 µF statt 15 µF
Ein Bauteil ist signifikant kleiner (heisst bei mir "AV", bei Dir "GK C49 AV")
Das daneben heisst "AG R3W" bei Dir "AG QK"
Dann sind alle 702 P (Mosfets?) bei Dir "72K"
Beim M-Bus Anschluss "84118 A3278" bei Dir "84118 4261"


Leider ist mein Wärmezähler noch nicht angekommen, sodass ich keine weiteren Tests machen kann.

Gruß Tom

![20240824_101824](https://github.com/user-attachments/assets/847d41df-27d7-441b-bf53-b6fff94c6240)
![20240824_102258](https://github.com/user-attachments/assets/a8b769c4-dd41-4c2f-907b-aa5e618d4fb5)
![20240824_102312](https://github.com/user-attachments/assets/3288a0a8-5a3d-48a6-b379-b5372b564e3e)

**Reply by Zeppelin500** [reply]
ich hab jetzt den Link zum Shop in der readme auf diesen hier angepasst

**Comment by emmrichd** [comment]
Hallo,

Ich habe hier bestellt:

https://a.aliexpress.com/_EyQhgIZ

Und das gleiche board wie Zeppelin 500 mit dem "np" Hersteller erhalten.

**Reply by Zeppelin500** [reply]
Vielen Dank! Hab gleich mal meinen Link auf der readme angepasst.

**Reply by Zeppelin500** [reply]
Das ist ja meine ursprüngliche Quelle! Wann hast du bestellt? Scheinbar haben die Reklamationen geholfen.

**Reply by emmrichd** [reply]
Am 26.9., am 4.10. angekommen 

Sind die m+ und m- Kontakte jeweils miteinander verbunden oder sind sie irgendwie paarweise zu nutzen?

**Reply by Zeppelin500** [reply]
Die sind jeweils gebrückt. Auf dem MBusino sind sie ebenfalls gebrückt und gehen gemeinsam auf die Klemmen.

**Comment by Maniacones** [comment]
Hallo. Der Thread ist ja nun schon ein wenig älter, aber ich habe das gleiche Problem mit der heißen Spule und dass nix geht. Gekauft hab ich das Teil im Juni dieses Jahr und kann's demnach auch nicht mehr umtauschen. Hatte früher keine Zeit gefunden für das Projekt und nun weiß ich nicht so recht wie ich weiter machen soll. 

1. auf Ali gibt's das bekannte Board mit der 150er Spule auf den Bildern, aber ob die dann auch geliefert wird?!
2. weiterhin auf Ali gibt's die Platine mit den abgewinkelten Steckern 
3. alternativ die Platine von Zihatec

Ich habe noch die alte Mbusino, also nicht Edition Z, da passt natürlich nur die Nummer 1 sauber rauf.

Bei der abgewinkelten und der Zihatec müsst ich irgendwas mit Kabeln zusammen schustern, das wäre als Notlösung schon ok bevor ich mir den nächsten Schrott bestelle. 

Geht die Zihatec den auch mit dem alten Mbusino oder bräuchte ich da auch Edition Z?

**Reply by Zeppelin500** [reply]
Nein, das passt mechanisch nicht. Für das Zihatec Modul brauchst du die Z Version.
PN?

**Comment by aut0mat3d** [comment]
Hi There!
At first: Thanks for Mbusino!

Starting with Mbusino today i faced the same Problem:
DykbRadio MBUS-TTL Module from Aliexpress is getting hot, no Voltage on M-Bus Rail,....
As i do own a IR Camera the Culprit was found in Seconds:

The Inductor and the 6-pin (?Step Up Driver?) SMD IC next to it are getting hot (Inductor reaches > 70°C on 5V capped to 200mA. 

After the Coil is a Diode (Marked as K16 on the Picture of the OP) which is inverse. (Manufacturing or Design Fault):
As this is a Step Up/Boost Coverter the Cathode of the Shottky Diode must not face towards the Inductor!
See: https://commons.wikimedia.org/wiki/File:Boost_converter.svg

Also, if you look at the Pictures from Zeppelin500 (first Reply) you can see the Diode after the Incutor facing away from it (this is the right Direction).

After rotating the Diode i do now have 30V DC on the M-bus Pins, the Converter draws roughly 30mA on 5V if i remember right.
This leaves me wondering, how big this bad Batch is, how many PCBs are shipped/returned, went to landfill,... - absolutely crazy!

Note: I am in the Process of Building Mbusino, fixing the Converter is the Step where i am at the Moment.
I dont know if the Converter is working in general,....

**Reply by kusmi** [reply]
nice find! - so, I'm really puzzled, there was not even a simple quality or "power-on" check from the manufacturer of that board…

**Reply by Zeppelin500** [reply]
Great! Thank you for the root cause!
Almost all replies about this Master was bad. I strongly recommend the Zihatec Master.

**Reply by Kutilak** [reply]
Thank you. Now i have same board with reversed diode. I solder it in the right way and I thing its OK. Is there some other issue with this board or after this repair its work OK ?

**Reply by aut0mat3d** [reply]
@ Kutilak 
I do have it working here with M-Bus and one Heatmeter (a Landis & Gyr T230) without any Problems.
DS18B12 or the I²¢ Bus not tested, but i dont see anything here what would cause issues.
One Detail: I mounted the TTL to M-Bus converter on the Backside of the PCB to acheive a better Reception on the HF-Side (Antenna dont gets "blocked/disturbed" by the Converter,.... - if you do so, double check the Placement as it is easy to mirror mont it,....

I would strongly suggest to attach the ESP via 2.54mm Sockets to remove it for the case you have to reflash it via USB as the ESP8266 uses the (only one available) Serial Port to communicate to the TTL to M-Bus converter,......

**Reply by Zeppelin500** [reply]
Just FYI, I'm sharing the PCBs.
